packages/next-templates/src/components/PopOver/PopOverButton.tsx

```diff
@@ -1,16 +1,50 @@
 import { Button } from '@utrecht/component-library-react';
-import React, { ForwardedRef, HTMLAttributes, PropsWithChildren, forwardRef } from 'react';
+import React, { ForwardedRef, HTMLAttributes, PropsWithChildren, MouseEvent, forwardRef } from 'react';
 
 type ExtraProps = HTMLAttributes<HTMLButtonElement> & {
   popoverTarget?: string;
   popoverTargetAction?: string;
   appearance?: string;
 };
 
+let check = true;
+
```

Comment:
Waarom is deze `check` nodig? Als het state is, dan heb ik liever dat je het opslaat via `targetEl.dataset.open` ofzo